packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -59,6 +59,8 @@ export type Token = {
 
 const DEFAULT_INTERVAL = 180000;
 
+export const DEFAULT_CACHE_REFRESH_THRESHOLD = 1000 * 60 * 60 * 24; // 24 hours
```

Comment:
I'm caching this supported chains data because its not the kind of data that will change very often.